Makefile

```diff
@@ -1,4 +1,14 @@
 # Dir where build binaries are generated. The dir should be gitignored
+
+## Set defaults
+export GO111MODULE := on
+export PATH := $(GOBIN):$(PATH)
```

Comment:
this is required in general for local go setup, do we need it as part of makefile, we can add this in readme ?

Reply:
`Makefile` is used in CI runs as well.